dashboard/src/pages/TreeDetails/Tabs/Build/BuildTab.tsx

```diff
@@ -254,7 +255,7 @@ const BuildTab = ({ treeDetailsData }: BuildTab): JSX.Element => {
             treeDetailsData={treeDetailsData}
           />
           {/* TODO Kill the CardGroups component once and for all */}
-          <CardsGroup cards={cards} />
+          <MemoizedErrorsSummaryBuild cards={cards} />
```

Comment:
Let's go simpler, as the name suggests `ErrorsSummaryBuild` is a specific component to be used as a summary card on tab build, it do not need to receive the `cards` array, only the `treeDetailsData?.archs` and the onClick functions